newsroom/news_api/__init__.py

```diff
@@ -22,5 +23,34 @@ def init_app(app):
         lazy_gettext('Image renditions the API can serve'),
         weight=600,
         description=lazy_gettext('A comma seperated list of the renditions that the API will return'),
-        default=app.config.get('NEWS_API_ALLOWED_RENDITIONS', '')
+        default=app.config.get('NEWS_API_ALLOWED_RENDITIONS', ''),
+        validator=validate_renditions
     )
+
+
+def is_safe_string(field, allowed_punctuation=("'")):
```

Comment:
Duplicate function, defined in `newsroom/utils.py`

Reply:
fixed!